webroot/src/pages/AuthCallback/AuthCallback.ts

```diff
@@ -39,15 +39,41 @@ export default class AuthCallback extends Vue {
         return this.$route.query?.code?.toString() || '';
     }
 
+    get userType(): string {
+        return this.$route.query?.state?.toString() || '';
+    }
+
     //
     // Methods
     //
     async getTokens(): Promise<void> {
         this.$store.dispatch('startLoading');
 
-        await this.getTokensStaff().catch(() => {
-            this.isError = true;
-        });
+        if (this.userType === AuthTypes.STAFF) {
+            await this.getTokensStaff().catch(() => {
+                this.isError = true;
+            });
+        } else if (this.userType === AuthTypes.LICENSEE) {
+            await this.getTokensLicensee().catch(() => {
+                this.isError = true;
+            });
+        } else {
+            let errorCount = 0;
+
+            await this.getTokensStaff().catch(() => {
+                errorCount += 1;
+            });
+
+            if (errorCount > 0) {
+                await this.getTokensLicensee().catch(() => {
+                    errorCount += 1;
+                });
+            }
+
+            if (errorCount > 1) {
```

Comment:
I suspect CSG will want a short comment here describing why the error count in this block matters.